docs/development/docs.rst

```diff
@@ -22,33 +22,37 @@ you can verify those changes through the documentation generated when you open a
 Contributing from your local machine
 ------------------------------------
 
-.. note::
-
-   You must use python 3.6 to generate the documentation.
-
 If you're making large changes to the documentation,
 you may want to verify those changes locally before pushing upstream.
 
 #. clone the `readthedocs.org` repository:
 
-   .. prompt:: bash
+   .. code-block:: bash
+
+      $ git clone --recurse-submodules https://github.com/readthedocs/readthedocs.org/
+
+#. create a virtual environment with Python 3.6 and activate it:
+
+   .. code-block:: bash
 
-      git clone --recurse-submodules https://github.com/readthedocs/readthedocs.org/
+      $ cd readthedocs.org
+      $ python3.6 -m venv .venv
+      $ source .venv/bin/activate
+      (.venv) $
```

Comment:
I would probably prefer a note here, or even a short explanation, stating that after the `$ source .venv/bin/activate` line the bash would get prefixed with `(.venv)` .
Changing this and the next `.. prompt::` blocks to `.. code-block::` has the disadvantage of the `$` sign now being selectable. Also, it is not immediately clear what is meant by the `(.venv) $` prefix.

Reply:
Aha! Ok, this got solved with recent changes!

Reply:
Ueah, right!